js/src/logger.ts

```diff
@@ -617,7 +617,7 @@ export class FailedHTTPResponse extends Error {
   public data: any;
 
   constructor(status: number, text: string, data: any = null) {
```

Comment:
nit: this is only called in one spot, maybe drop the default value

Reply:
good call